src/Frame.js

```diff
@@ -0,0 +1,59 @@
+function Frame(frameNum) {
+  this.rollCounter = 0;
+  this.rolls = [];
+
+  this.strike = false;
+  this.spare = false;
+
+  this.currentScore = 0;
+  this.frameNum = frameNum;
+}
+
+Frame.prototype.roll = function roll(pinsKnocked) {
+  this.isRollLegit();
+  this.isStrike(pinsKnocked);
+
+  if (this.strike !== true) {
+    this.rolls.push({ roll: parseInt(`${pinsKnocked}`, 10) });
+    this.calculateScore(pinsKnocked);
+    this.isSpare();
+  }
+
+  return pinsKnocked;
+};
+
+// PRIVATE METHODS
+
+Frame.prototype.isRollLegit = function isRollLegit() {
+  const newCounter = this.rollCounter + 1;
+
+  if (this.strike === false && this.spare === false && this.frameNum === 10 && newCounter === 3) {
```

Comment:
Sometimes if I'm checking against a long list of conditions like this I like to create a descriptive variable name to make things a bit more readable.  So something like

var isNotStrikeOrSpareIsFinalFrameFinalRoll = this.strike === false && this.spare === false && this.frameNum === 10 && newCounter === 3

if (isNotStrikeOrSpareIsFinalFrameFinalRoll) {
    throw new Error('The game is over!');
}

This is just a personal preference, but I think it helps to explain what your code is trying to do.